libcovebods/schema.py

```diff
@@ -4,14 +4,14 @@
 class SchemaBODS(SchemaJsonMixin):
 
     def __init__(self, json_data=None, lib_cove_bods_config=None):
-        self.lib_cove_bods_config = lib_cove_bods_config
+        self.config = lib_cove_bods_config
         if isinstance(json_data, list) and len(json_data) > 0:
             self.schema_version, self.pkg_schema_url, self.schema_host = \
                 self.get_schema_version_of_statement(json_data[0])
         else:
-            self.pkg_schema_url = lib_cove_bods_config.config['schema_url']
-            self.schema_host = lib_cove_bods_config.config['schema_url_host']
-            self.schema_version = lib_cove_bods_config.config['schema_version']
+            self.pkg_schema_url = config.config['schema_url']
```

Comment:
I doesn't look like you've renamed the method param that this used to come from, so should this be `self.config`? 

Aside: `flake8` seems to have picked this up, but I'm surprised that the unit tests look to have passed - do they not exercise this bit of the code?

Reply:
The tests in this repo all set up the config the same way, so they don't hit this bit of code. Running the cove-bods tests with the new lib-cove-bods will though.